pkg/scheduler/metrics/metrics.go

```diff
@@ -28,13 +28,12 @@ import (
 const (
 	// SchedulerSubsystem - subsystem name used by scheduler
 	SchedulerSubsystem = "scheduler"
-	// Below are possible values for the operation label. Each represents a substep of e2e scheduling:
+	// Below are possible values for the operation label.
```

Comment:
The label is actually called `work`. Although I actually like `operation` better, but that can't be changed.

Reply:
https://github.com/kubernetes/kubernetes/issues/112034

Reply:
👍